### DISCUSSION TITLE
Catch domain and redirect to sub path

### DISCUSSION THREAD

**Discussion by jimmi-rbn** [vercel/next.js#31434] [Ideas]
### Describe the feature you'd like to request

If you set multiple domains in vercel - be able to catch that domain and redirect it to a specific sub path

### Describe the solution you'd like

`async rewrites() {

        return [
            {
                source: "/",
                has: [
                    {
                        type: "header",
                        key: ":authority", // or request url or similar
                        value: "example.cz",
                    },
                ],
                destination: "/cs-CZ/",
            },
        ];
    },`

### Describe alternatives you've considered

Making a middle-layer that could catch the request url

**Comment by timneutkens** [comment]
Have you had a look at https://nextjs.org/docs/middleware? It allows for more flexibility in routing rules.
